cypress/integration/set-value.ts

```diff
@@ -1,20 +1,44 @@
 /// <reference types="cypress" />
 
+function getHtmlWithAttributesSorted(html: string): string {
+  return html
+    .replace(/\/?unicorn\.\w+\./, 'unicorn.')
+    .replace(/<(\w+)([^>]+)>/g, ($0, $1, $2) => {
+      const regex = /([a-zA-Z0-9_-]+)="([^"]+)?"/g;
+      const attributes = [];
+      let matches;
+
+      while ((matches = regex.exec($2))) {
+        attributes.push({ key: matches[1], value: matches[2] || '' });
+        console.log(matches[1], matches[2]);
```

Comment:
I imagine we do not want this console.log even though it is tests.

Reply:
Nice catch, fixed on 8603f5b.